shell/browser/api/electron_api_session.cc

```diff
@@ -976,6 +977,46 @@ v8::Local<v8::Value> Session::GetPath(v8::Isolate* isolate) {
   return gin::ConvertToV8(isolate, browser_context_->GetPath());
 }
 
+void Session::SetCodeCachePath(gin::Arguments* args) {
+  base::FilePath code_cache_path;
+  auto* storage_partition = browser_context_->GetDefaultStoragePartition();
+  auto* code_cache_context = storage_partition->GetGeneratedCodeCacheContext();
+  if (code_cache_context) {
+    if (!args->GetNext(&code_cache_path) || !code_cache_path.IsAbsolute()) {
+      args->ThrowTypeError(
+          "Absolute path must be provided to store code cache.");
+      return;
+    }
+    code_cache_context->Initialize(
+        code_cache_path, 0 /* allows disk_cache to choose the size */);
```

Comment:
Would there be any benefit in exposing this to let users choose the size?

Reply:
Maybe ? At the moment there is no use case to limit the cache size, can revisit if there are valid scenarios.